.taskcluster.yml

```diff
@@ -57,7 +57,7 @@ tasks:
     - taskId: {$eval: as_slugid("docker_build")}
       dependencies:
         - {$eval: as_slugid("code_checks")}
-      provisionerId: aws-provisioner-v1
+      provisionerId: proj-relman
       workerType: releng-svc
```

Comment:
Can we also use `workerType: ci` here ? We only need a workerType with the privileged status available to build a docker image

Reply:
Ah yeah! I totally missed that workertype.